src/components/LinearGradient/index.js

```diff
@@ -0,0 +1 @@
+export {LinearGradient} from "./LinearGradient"
```

Comment:
can you explain how this works? So this index file essentially exports either `.js` if it's a mobile paltform or `.web.js` if it's web? Which library handles the logic for this?

Reply:
See comment below